### DISCUSSION TITLE
DBMS-Chp1&2&3 · Cup Air

### DISCUSSION THREAD

**Discussion by ACupofAir** [ACupofAir/ACupofAir.github.io#21] [Announcements]
https://acupofair.github.io/posts/dbms/chp1_2_3/ 

 
数据库管理系统第一，二，三章笔记

**Comment by LeiBuDao** [comment]
绝绝子

**Comment by ACupofAir** [comment]
感谢
> @LeiBuDao
> 绝绝子

感谢润神，压中一半以上的题，所有的sql语句，外加一道陈述题🐮
